CMakeLists.txt

```diff
@@ -131,7 +131,9 @@ else (APPLE)
 endif(APPLE)
 
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
+if (NOT CMAKE_SKIP_RPATH) # Option to not set rpath - required for packaging on Debian
```

Comment:
Perhaps it is not necessary here, but it is certainly necessary when the compiler flags are set manually on [line 414](https://github.com/shoogle/MuseScore/blob/debian-packaging/CMakeLists.txt#L414) of the same file (hence my changes there) and I thought that having the IF statement everywhere makes things clearer for package maintainers.

I never tried it without the IF statement around `set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)` (my starting point for this change was tsmithe's patch in this [old thread](http://dev-list.musescore.org/Setting-rpath-in-mscore-binary-td4112914.html)). I can try it without the IF statement around `set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)` if you prefer?